toolkit/tools/imagecustomizerschemacli/main.go

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/invopop/jsonschema"
+	"github.com/microsoft/azurelinux/toolkit/tools/imagecustomizerapi"
+)
+
+func main() {
+	if len(os.Args) < 1 {
+		fmt.Println("Usage: jsonschemacli <output_file>")
+		os.Exit(1)
+	}
+
+	reflector := jsonschema.Reflector{}
+	reflector.RequiredFromJSONSchemaTags = true
+	schema := reflector.Reflect(&imagecustomizerapi.Config{})
+
+	schemaJSON, err := json.MarshalIndent(schema, "", "  ")
+	if err != nil {
+		log.Fatalf("Failed to marshal schema: %v", err)
```

Comment:
You should move most of the logic into a helper function that returns a normal `error`. Then handle the `log.Fatalf` in a single place.